File: src/main/java/com/tao/guess/Gamefor.java
package com.tao.guess;

import java.util.Random;
import java.util.Scanner;

public class Gamefor {
    public static void main(String[] args) {

        int secret=new Random().nextInt(10)+1;
        System.out.println(secret);

        Scanner scanner=new Scanner(System.in);

        for(int i=1;i<=4;i++){
           System.out.println("please enter a number:"+i+"/4");
           int number=scanner.nextInt();
           System.out.println("the no."+i+"time:  "+number);
           if(number>secret){
               System.out.println("please be higher");
           }else if(number < secret){
               System.out.println("please be lowwer");
           }else{
               System.out.println("you get it:  "+number);
               break;
           }
            System.out.println("end");
        }

    }
}

File: src/main/kotlin/com/tao/com/Gamefor.kt
package com.tao.com

import java.util.*

fun main(args: Array<String>) {
    val secret=Random().nextInt(10)+1
    println(secret)
    val scanner=Scanner(System.`in`)
    for(i in 1..4){
        System.out.println("請輸入一數字(${i}/4:)")
        var  number=scanner.nextInt()

        println("第${i}次輸入: $number")

        if(number>secret){
            println("be lowwer")
        }else if(number<secret){
            println("be higher")
        }else{
            println("you get it:"+number)
            println("game over")
            break

        }
    }

}
